Update cms-element-form-privacy.html.twig - make privacyUrl and tosUr…

bitranox opened the pull request:
Shopware Issue tracker Ticket: NEXT-17851 - privacyUrl and tosUrl not working
### 1. Why is this change necessary?
privacyUrl and tosUrl not working

### 2. What does this change do, exactly?
fix the Error

### 3. Describe each step to reproduce the issue or behaviour.


### 4. Please link to the relevant issues (if any).
Shopware Issue tracker Ticket: NEXT-17851

bitranox committed:
Update cms-element-form-privacy.html.twig - make privacyUrl and tosUrl work

Shopware Issue tracker Ticket: NEXT-17851 - privacyUrl and tosUrl not working

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/shopware/platform?pullRequest=2127) <br/>All committers have signed the CLA.

shyim commented on src/Storefront/Resources/views/storefront/element/cms-element-form/form-components/cms-element-form-privacy.html.twig:
```diff
@@ -17,9 +17,14 @@
         {% block cms_form_privacy_opt_in_label %}
             <label for="{{ identifierTemplate|format(_key) }}" class=" custom-control-label">
                 {{ "general.privacyNotice"|trans({
-                    '%url%': path('frontend.cms.page',{ id: config('core.basicInformation.privacyPage') })
```
We need still to replace %url% as someone should adjusted that to that value in the translation

shyim commented:
Hey,

Can you add a changelog file according to this ADR? https://github.com/shopware/platform/blob/trunk/adr/2020-08-03-Implement-New-Changelog.md

bitranox committed:
Update cms-element-form-privacy.html.twig

bitranox committed:
Create 2021-10-13-set-privacyUrl-and-tosUrl.md

add changelog

bitranox committed:
Update cms-element-form-privacy.html.twig

bitranox committed:
Update cms-element-form-privacy.html.twig

added comments and re-arrange formatting to make it more readable

bitranox committed:
Formatting and Comments

added comments and re-arrange formatting to make it more readable - again. The checker urges me to use a new commit message ... grrrr.

bitranox committed:
update comments again to pass tests

update comments again

bitranox committed:
update typos to pass test

update typos to pass test

bitranox committed:
update comments to pass tests 

update comments to pass tests

bitranox commented:
ok, it still complains about "You should not use the same commit message multiple times" - 
and I can not remove that. Someone can take care please, i am finished.

NiklasLimberg commented:
Hi @bitranox,
I'll take care of this PR, consider it merged

shyim commented:
Thanks for your Contribution! :blue_heart:
